--- davical-0.9.9.2/inc/always.php.in.orig	2010-09-22 15:19:34.000000000 +0200
+++ davical-0.9.9.2/inc/always.php.in	2010-09-22 15:19:48.000000000 +0200
@@ -70,22 +70,8 @@
 
 // Utilities
 if ( ! @include_once('AWLUtilities.php') ) {
-  $try_paths = array(
-        '../../awl/inc'
-      , '/usr/share/awl/inc'        // Where it ends up on Debian
-      , '/usr/share/php/awl/inc'    // Fedora's standard for PHP libraries
-      , '/usr/local/share/awl/inc'
-  );
-  foreach( $try_paths AS $awl_include_path ) {
-    if ( @file_exists($awl_include_path.'/AWLUtilities.php') ) {
-      set_include_path( $awl_include_path. PATH_SEPARATOR. get_include_path());
-      break;
-    }
-  }
-  if ( ! @include_once('AWLUtilities.php') ) {
-    echo "Could not find the AWL libraries. Are they installed? Check your include_path in php.ini!\n";
-    exit;
-  }
+  echo "Could not find the AWL libraries. Are they installed? Check your include_path in php.ini!\n";
+  exit;
 }
 
 // Ensure that ../inc is in our included paths as early as possible
--- davical-0.9.9.2/dba/create-database.sh~	2010-09-21 03:40:44.000000000 +0200
+++ davical-0.9.9.2/dba/create-database.sh	2010-09-22 17:02:18.000000000 +0200
@@ -16,16 +16,10 @@
 
 #
 # Attempt to locate the AWL directory
-AWLDIR="${DBADIR}/../../awl"
+AWLDIR="/usr/share/php/awl"
 if ! testawldir "${AWLDIR}"; then
-  AWLDIR="/usr/share/awl"
-  if ! testawldir "${AWLDIR}"; then
-    AWLDIR="/usr/local/share/awl"
-    if ! testawldir "${AWLDIR}"; then
-      echo "Unable to find AWL libraries"
-      exit 1
-    fi
-  fi
+  echo "Unable to find AWL libraries"
+  exit 1
 fi
 
 export AWL_DBAUSER=davical_dba
