--- davical-0.9.9.2/inc/always.php.in~	2010-09-21 04:28:04.000000000 +0200
+++ davical-0.9.9.2/inc/always.php.in	2010-09-22 13:46:31.000000000 +0200
@@ -71,9 +71,7 @@
 // Utilities
 if ( ! @include_once('AWLUtilities.php') ) {
   $try_paths = array(
-        '../../awl/inc'
-      , '/usr/share/awl/inc'
-      , '/usr/local/share/awl/inc'
+        '@PHP_DATA_DIR@/awl/inc'
   );
   foreach( $try_paths AS $awl_include_path ) {
     if ( @file_exists($awl_include_path.'/AWLUtilities.php') ) {
